include/openssl/pemerr.h

```diff
@@ -11,9 +11,7 @@
 #ifndef HEADER_PEMERR_H
 # define HEADER_PEMERR_H
 
-# ifndef HEADER_SYMHACKS_H
-#  include <openssl/symhacks.h>
-# endif
+# include <openssl/symhacks.h>
```

Comment:
again, funny, this was probably cherry-picked, I see it in several places, but `make update` reverts it.

Reply:
make update does not result in any changes for me in the 1.1.1-stable branch

Reply:
yes, confirmed. But the script needs a trigger to do something.

Reply:
try `make update ERROR_REBUILD=-rebuild`

Reply:
I do get that change too after applying your patch